javascript/ql/src/Security/CWE-078/UnsafeShellCommandConstruction.qhelp

```diff
@@ -26,6 +26,16 @@
 		interpretation by the shell.
 	</p>
 
+	<p>
+
+		Note, however, that if you are given the arguments as a single string,
+		it is not safe to simply split the string on whitespace, since an
+		argument may contain quoted whitespace which would cause it to be split
+		into multiple arguments. Instead, use a library such as
+		<code>shell-quote</code> to parse the string into an array of arguments.
```

Comment:
```suggestion
If given arguments as a single string, avoid simply splitting the string on 
whitespace. Arguments may contain quoted whitespace, causing them to split into 
multiple arguments. Use a library like <code>shell-quote</code> to parse the string 
into an array of arguments instead.
```

The same GPT-4 comment. 

(I asked GPT-4 about each of the QHelps in a single chat session, it gave this suggestion on all).